packages/manager/apps/pci-public-ip/public/translations/common/Messages_fr_FR.json

```diff
@@ -50,5 +50,6 @@
   "pci_additional_ips_country_NL": "Pays-Bas",
   "pci_additional_ips_country_PT": "Portugal",
   "pci_additional_ips_country_SG": "Singapour",
-  "pci_additional_ips_country_IN": "Inde"
+  "pci_additional_ips_country_IN": "Inde",
+  "common_click_here_btn": "Cliquez ici"
```

Comment:
Looks like all keys start with `pci_additional_ips_`, so I'll suggest to do the same for this one :)